react/src/hooks/backendai.tsx

```diff
@@ -73,16 +73,18 @@ export const useResourceSlotsDetails = (resourceGroupName?: string) => {
     queryFn: () => {
       // return baiClient.get_resource_slots();
       if (
-        _.isEmpty(resourceGroupName) ||
```

Comment:
Why did you change this line? lodash `isEmpty` can check empty string, null, undefined, etc.

Reply:
This was fixed by @yomybaby , so I don't know exactly why.

Reply:
`resourceGroupName` is a string. !resourceGroupName is enough to determine if it is empty and better for TypeScript inference.